### ISSUE TITLE
Advice on specifying different distributions for different channels

### ISSUE THREAD

**Issue by Alcampopiano** [pymc-labs/pymc-marketing#485]
Hello! Thanks for the fantastic library. Just a basic question for you:

Suppose I have two channels, one which I want to associate with a uniform prior, and one with a truncated normal. 

I tried using syntax like this but it is not accepted by `mmm.fit`:

```python
"beta_channel": 
   {
     'channel_1': {'dist': 'TruncatedNormal', 'kwargs': {'sigma': 1, "mu": 1, "lower": 0}},
     'channel_2': {'dist': 'uniform', 'kwargs': {'lower': 0, 'upper': 1}}
   }
```

Is this possible to do in PyMC-marketing?

Any nudge in the right direction would be much appreciated!

**Comment by ricardoV94** [comment]
I don't think you can do that, but you could use vector parameters to get different priors under the same family, like

`{'dist': 'TruncatedNormal', kwargs': {'sigma': [1, 100]', 'mu': [1, 0.5], 'lower': 0, 'upper: [np.inf, 1]'}}`

Which will make the first channel like you had before, and the second something "uniformish" looking (you can make it more or less uniform by changing the second sigma)

**Comment by Alcampopiano** [comment]
This is great thank you!
